bitnami/redis/Chart.yaml

```
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 15.0.3
+version: 15.0.4
```

it's a new feature, could you update the version to 15.1.0?
```suggestion
version: 15.1.0
```